Mathlib/Data/Quot.lean

```diff
@@ -451,67 +451,177 @@ Note that in later files the preferred spelling is `⊤ : Setoid α`. -/
 def trueSetoid : Setoid α :=
```

Comment:
This is only ever used to define `Trunc`, and the docstring explains there's a better spelling available, so we should just deprecate it as well.